### DISCUSSION TITLE
DOMException: Failed to execute 'setAttribute' on 'Element'

### DISCUSSION THREAD

**Discussion by ryangjchandler** [livewire/livewire#3101] [7. Bugs (archived)]
Couldn't think of a good issue title, so figured the error itself would provide some good searchability.

### Description

When Livewire returns the response and attempts to patch the DOM, it throws the following exception:
![image](https://user-images.githubusercontent.com/41837763/100214540-efd62f00-2f07-11eb-8898-4428c8ae26a2.png)

The action is being called inside of an Alpine `@click.prevent` directive, using the new `await` syntax. Essentially this:

```html
<button @click.prevent="await $wire.callAction()"></button>
```

Upgrading to latest Livewire version doesn't fix the problem either.

Switching to the `x-on:` syntax fixes the problem. Just seems to be a problem with the shorthand syntax.

### Exact steps to reproduce

Not even sure about the steps to reproduce, as I've never seen it before and can't reproduce the problem in Playground.

### Stripped-down, copy-pastable code snippets

**Context**
- Livewire version: 2.3.1
- Laravel version: 8.15.0
- Alpine version: 2.7.x
- Browser: Chrome

**Comment by geechartier** [comment]
I have a similar error with `@click` when a Livewire event causes a component to be re-rendered. Likewise, switching to `x-on` syntax solves the problem. It's a shame I decided to standardize on the `@` syntax. I will now switch to `x-on` syntax throughout.

- Livewire version: 2.4.4
- Laravel version: 8.41.0
- Alpine version: 2.8.2
- Browser: Chrome

**Comment by calebporzio** [comment]
Should be fixed on the master branch: https://github.com/livewire/livewire/pull/3152

please verify
